src/services/goals.js

```diff
@@ -984,21 +984,56 @@ export async function goalsForGrants(grantIds) {
   });
 }
 
+async function destroyActivityReportObjectiveMetadata(activityReportObjectiveIdsToRemove) {
+  return Promise.all([
+    ActivityReportObjectiveFile.destroy({
+      where: {
+        activityReportObjectiveId: activityReportObjectiveIdsToRemove,
+      },
+    }),
+    ActivityReportObjectiveResource.destroy({
+      where: {
+        activityReportObjectiveId: activityReportObjectiveIdsToRemove,
+      },
+    }),
+    ActivityReportObjectiveTopic.destroy({
+      where: {
+        activityReportObjectiveId: activityReportObjectiveIdsToRemove,
+      },
```

Comment:
Should this be in the other file with the rest of the caching code?

Reply:
sure, can be